src/libraries/Common/src/Interop/Windows/Kernel32/Interop.Constants.cs

```diff
@@ -2,12 +2,18 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-internal partial class Interop
+internal static partial class Interop
 {
-    internal partial class Kernel32
+    internal static partial class Kernel32
```

Comment:
I have unified a few duplicate `Interop.*` files, but left most of the unification for separate PR so that it is easier to review.